setup/01_optional-python-setup-preferences/native-uv.md

```diff
@@ -53,26 +53,18 @@ powershell -c "irm https://astral.sh/uv/install.ps1 | more"
 > For more installation options, please refer to the official [uv documentation](https://docs.astral.sh/uv/getting-started/installation/#standalone-installer).
 
 &nbsp;
-## 2. Install Python
+## 2. Install Python packages and dependencies
 
-You can install Python using uv:
+To install all required packages from a `pyproject.toml` file (such as the one located at the top level of this GitHub repository), run the following command, assuming the file is in the same directory as your terminal session:
 
 ```bash
-uv python install 3.10
+uv sync --dev --python 3.11
```

Comment:
Thanks for the PR! Question: Doesn't `uv sync` remove packages that are not explicitely defined as dependencies in the `pyproject.toml` though? Sure, in some cases that's maybe what you want, but I can also see where this is annoying if you

1) setup up your new venv, install a bunch of packages
2) then run `uv sync` on the pyrpoject.toml and discover those are then gone?

Reply:
That's correct. 

My read of this instructions was that they're for setting up the project environment for the first time (it doesn't, for example, include instructions on setting up a venv). 

If you want to prevent that, you can do `uv sync --inexact`.

But beyond that, if you're using pyproject.toml to management dependencies, it's an antipattern to add dependencies to the environment that you're not tracking in the pyproject.toml. `uv add` and `uv pip install` shouldn't be used together.

One last point, `uv sync` is going to explicitly tell you if it installs packages, so it should at least be clear to end users what is going on.